plugin-maven/src/main/java/com/diffplug/spotless/maven/json/JacksonJson.java

```diff
@@ -44,12 +44,16 @@ public class JacksonJson implements FormatterStepFactory {
 	@Parameter
 	private Map<String, Boolean> jsonFeatures = Collections.emptyMap();
```

Comment:
this could be deprecated and replaced by jsonGeneratorFeatures for clarity

Reply:
I have strong preference against API churn.